### DISCUSSION TITLE
Transpose weekly time tracking table

### DISCUSSION THREAD

**Discussion by codetalker78** [kromitgmbh/titra#75] [Ideas]
The week view is a nice timesheet view that allows efficient time documentation. But currently, the tab order moves to the next day of the same project:
Project A Monday --> Project A Tuesday ---> Project A Wednesday ...

This requires the user to enter time for one project for each day, before revisiting each day for the second project and so on. This is not how a person thinks about their work week. It would be more intuitive, if the user would be able to step through all projects on one day before switching to the next day and stepping through all projects again.

This could be achieved with a change in the tab order, or by transposing the table: one column per project, one row per day.
If the project list grows large, this could become an issue with screen width, but on the other hand, this would allow to add more than seven days to this type of time-sheet view. A monthly table would be useful too and a very simple addition

**Comment by faburem** [comment]
I  think there are different use cases depending on how you track time. In our company for example we work on multiple different projects with many long-term tasks (think about maintenance or support activities). For such scenarios the current setup works best because you divide your time for the week i.e. 20% support for Project A for the whole week, 40% maintenance on Project B and so on.
However I totally see the other use case as well and would love to investigate this design option further, maybe not changing the current view though but providing it as an alternative option for the week view. I will add all these suggestions to our Roadmap - thanks a lot for your great feedback!

**Reply by faburem** [reply]
This feature is available in the latest version of titra, it is called "Timeline" view.
